src/datetimepicker.test.jsx

```diff
@@ -218,4 +216,30 @@ describe('DateTimePicker', function() {
 		});
 
 	});
+
+	describe('onChangeCallback in props', () => {
+		let changeCallbackSpy;
+
+		beforeEach(() => {
+			changeCallbackSpy = jest.fn();
```

Comment:
the conventional way to use `jest.fn` is to assign it once, and then call [`mockFn.mockClear()`](https://facebook.github.io/jest/docs/mock-function-api.html#mockfnmockclear) each time you want to re-test it in a test suite.